.travis.yml

```diff
@@ -3,6 +3,10 @@ sudo: required
 services:
```

Comment:
It was there from the beginning: https://github.com/dlang-tour/core-rdmd/blob/master/.travis.yml